lib/application.js

```diff
@@ -54,6 +55,16 @@ class BookkeepingApplication {
             if (gRPCOrigin) {
                 await this.gRPCServer.listen(gRPCOrigin);
             }
+
+            // Start background housekeeping
+            setInterval(async () => {
+                try {
+                    await handleLostRunsAndEnvironments();
+                    this.logger.debug('Updated lost runs and environments');
+                } catch (error) {
+                    this.logger.error(`Error while handling lost logs: ${error}`);
+                }
+            }, 15 * 60 * 1000);
```

Comment:
I would say 15 minutes will for sure impact the statistics.
Initially, we agree on 15 seconds but to be on the safe side, perhaps we start with 30 seconds?

Reply:
I misunderstood, I also thought it was a lot... Let's try with 30 seconds yes